src/components/SearchBar.js

```diff
@@ -0,0 +1,23 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+const propTypes = {
+  onSearch: PropTypes.func.isRequired,
+  search: PropTypes.string.isRequired,
+};
+
+function SearchBar({ onSearch, search }) {
```

Comment:
Why not arrow function?

Reply:
rewrote all components using arrow function.